libpod/networking.go

```diff
@@ -38,7 +38,8 @@ func (r *Runtime) createNetNS(ctr *Container) (err error) {
 
 	podNetwork := getPodNetwork(ctr.ID(), ctr.Name(), ctrNS.Path(), ctr.config.PortMappings)
 
-	if err := r.netPlugin.SetUpPod(podNetwork); err != nil {
+	_, err = r.netPlugin.SetUpPod(podNetwork)
+	if err != nil {
```

Comment:
nit: Still can be on single line.

Reply:
We're going to need the first return (CNI result with IP address and other info) eventually, so I figured I would put it on a new line